frontend/src/metabase-lib/parameters/utils/cards.ts

```diff
@@ -42,6 +43,9 @@ export function getCardUiParameters(
       };
     }
 
-    return { ...parameter, hasVariableTemplateTagTarget: true };
+    return {
+      ...parameter,
+      hasVariableTemplateTagTarget: !isDimensionTarget(target),
```

Comment:
Fix from `master`. In case we fail to find the field, do not assume it's a native query variable.